<!DOCTYPE HTML>
<html lang="en-US">
<head>
	<meta charset="UTF-8">
	<meta http-equiv="x-ua-compatible" content="ie=edge">
	<title>iDEC 2025 | CPU_CHINA</title>
	<meta name="description" content="">
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<!-- bootstrap CSS -->
	<link rel="stylesheet" href="css/bootstrap.min.css" type="text/css" media="all">
	<!-- carousel CSS -->
	<link rel="stylesheet" href="css/owl.carousel.min.css" type="text/css" media="all">	
	<!-- responsive CSS -->
	<link rel="stylesheet" href="css/responsive.css" type="text/css" media="all">
	<!-- animate CSS -->
	<link rel="stylesheet" href="css/animate.css" type="text/css" media="all">	
	<!-- animated-text CSS -->
	<link rel="stylesheet" href="css/animated-text.css" type="text/css" media="all">	
	<!-- font-awesome CSS -->
	<link rel="stylesheet" href="css/all.min.css" type="text/css" media="all">	
	<!-- font-flaticon CSS -->
	<link rel="stylesheet" href="css/flaticon.css" type="text/css" media="all">	
	<!-- theme-default CSS -->
	<link rel="stylesheet" href="css/theme-default.css" type="text/css" media="all">	
	<!-- meanmenu CSS -->
	<link rel="stylesheet" href="css/meanmenu.min.css" type="text/css" media="all">
	<!-- Main Style CSS -->
	<link rel="stylesheet" href="css/style.css" type="text/css" media="all">
	<!-- transitions CSS -->
	<link rel="stylesheet" href="css/owl.transitions.css" type="text/css" media="all">
	<!-- venobox CSS -->
	<link rel="stylesheet" href="css/venobox.css" type="text/css" media="all">
	<!-- widget CSS -->
	<link rel="stylesheet" href="css/widget.css" type="text/css" media="all">
	<!-- Jquery UI CSS -->	
	<link rel="stylesheet" href="css/jquery-ui.min.css" type="text/css" media="all">
	<!-- ScrollCue CSS -->	
	<link rel="stylesheet" href="css/scrollCue.css" type="text/css" media="all">
	
	<link rel="stylesheet" href="css/dark.css" type="text/css" media="all">

	<!-- modernizr js -->	
	<script src="js/modernizr-3.5.0.min.js"></script>

	<link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
</head>
<body>
		

<div class="da" id="toggleButton">
</div>
<!--===================start Data Header  Area====================-->
<div class="header-area" id="sticky-header">
	<div class="container-fluid">
		<div class="row align-items-center d-flex">
			<div class="col-lg-3">
				<div class="header-logo">
					<a class="main-logo" href="index.html"><img src="image/logo_white.png" alt=""></a>
					<a class="stiky-logo" href="index.html"><img src="image/logo_transp.png" alt=""></a>
				</div>
			</div>
			<div class="col-lg-9">
				<nav class="organic_menu">
					<div class="header-menu">
						<ul class="nav_scroll">
							<li class="gap"><a href="index.html">Home</a></li>
							<li class="gap"><a href="members.html">  Team <span><i class="fas fa-angle-down"></i></span></a>
								<div class="sub-menu">
										<ul>
											<li><a href="members.html">Members</a></li>
										</ul>
									</div>
							<li class="gap"><a href="background.html"> Project <span><i class="fas fa-angle-down"></i></span></a>
								<div class="sub-menu">
										<ul>
											<li><a href="background.html">Background</a></li>
											<li><a href="result.html">Result</a></li>
											<li><a href="files/idec25.pdf">Report</a></li>
											<li><a href="files/idec25_si.pdf">Supplementary Information</a></li>
										</ul>
									</div>
							<li class="gap"><a href="Protocol.html">Documentation <span><i class="fas fa-angle-down"></i></span></a>
								<div class="sub-menu">
									<ul>
										<li><a href="Protocol.html">Protocols</a></li>
										<li><a href="notebook.html">Lab Notebook</a></li>
									</ul>
								</div>
							</li>
								</div>
							</li>
						</ul>
					</div>
				</nav>
			</div>
		</div>
	</div>
</div>
<!-- ================mobile menu seection====== -->
<!-- Mobile Menu Section -->
<div class="mobile-menu-area d-sm-block d-md-block d-lg-none">
	<div class="mobile-menu">
	  <nav class="menu-box">
		<div class="nav-logo">
		  <a href="index.html">
			<img src="image/logo_transp.png" alt="Logo" title="Logo">
		  </a>
		</div>
  
		<!-- Menu will be populated by jQuery -->
		<ul class="navigation clearfix"></ul>
	  </nav>
	</div>
  </div>
<!--========================end mobile menu area==========================-->
<!-- ==========Hero section start ===================== -->
	<div class="bratecome-section">
		<div class="container">
			<div class="contant reveal-rotate">
				<div class="title">
					<h2>Result</h2>
					<br>
				</div>
				<div class="inner-contant">
					<ul>
						<li>Project<i class="fa fa-angle-right" aria-hidden="true"></i> Result</li>
					</ul>
				</div>
			</div>
		</div>
	</div>
<!-- ===========Hero section end ===================== -->


<!-- Start Service detail Section  -->
	<!--==================================================-->
	<div class="service-details">
		<div class="container">
			<div class="custom-row">
				<div class="custom-col-left">
					<aside class='left-nav-outside'>
						<div class="nav-sidebar" id="maole">
							<h2>Result</h2>
							<ul class="nav-list">
								<li class="nav-item"><a href="#Expression" class="nav-link">Expression and Purification of Enzyme Variants</a ></li>
								<li class="nav-item"><a href="#Kinetic" class="nav-link">Kinetic characteristics of hydrolase and ligase</a ></li>
								<li class="nav-item"><a href="#Ligase" class="nav-link">Engineering a Ligase Requires Additional Information</a ></li>
								<li class="nav-item"><a href="#Dis" class="nav-link">Discussions</a ></li>
								<li class="nav-item"><a href="#Bibli" class="nav-link">Bibliography</a ></li>
							</ul>
						</div>
					</aside>
				</div>

				<!-- 右侧内容列（替代 col-lg-10） -->
				<div class="custom-col-right">
					<div class="service-dtls-all-items">
						<div class="sevice-dtls-content">
							<a name = 'Expression'><h2>Expression and Purification of Enzyme Variants</h2></a>
							<br>
							<p>
								To investigate the effects of site-directed mutagenesis, the wild-type (WT) and three mutant
enzymes (Design1, Design2_IM(The intermediate construct from Design 2) and Design2) were
expressed and purified. Quantification of the purified proteins revealed that the mutations significantly impacted expression levels. The WT enzyme exhibited a robust expression yield of 3.27
mg/L. In stark contrast, all three mutants showed a substantial reduction in yield. The final yields for
Design2_IM, Design1, and Design2 were 0.788mg/L, 1.32 mg/L, and 1.08 mg/L, respectively. These
levels correspond to approximately 24%, 40%, and 33% of the WT expression level.
							</p>
							<div class="imgbox"><img src="image/reportimg/Presentation3_17.png" alt=""></div>
							<div class="caption">
            <p><strong>Figure 3. </strong>Characterization of the two ligation systems within a 4 Å proximity.</p>
        </div>


							<a name= 'Kinetic'><h2>Kinetic characteristics of hydrolase and ligase</h2></a>
							<br>
							<p>
							To elucidate the key determinants of the covalent modifications on reactivity, the hydrolytic and
ligation activities of several enzyme systems were evaluated (summarized in <a href="#t2">Table 2</a>). Herein, the
acyl donor refers to the portion of the hydrolysis substrate that remains covalently attached to the
catalytic residue following the formation of the ester intermediate, whereas the acyl acceptor is the
ligation substrate. <a href="#f4">Figure 4</a> shows the distribution of the attack angle between water molecules and
the ester intermediate across the five simulated hydrolytic reaction systems. Perhaps due to the
significant deviation of the initial structure from the native binding pose, the simulated ligation
systems of Bu2g(V/GA) and Trypsiligase exhibited mutual approximation between the N-terminus
of the acyl-acceptor and the electrophilic center of the acyl-donor. Furthermore, the resulting attack
angles and attack dihedrals exhibited a defined distribution pattern.
							</p>
							<div class="imgbox"><img src="image/reportimg/Presentation3.png" alt=""></div>
							<div class="caption">
            <p><strong><a name= 'f4'></a>Figure 4. </strong>Statistical Distribution of the local Water-Oxygen Attack Angle within 3.5 Å of the Ester
Intermediate for Four Wild-Type Hydrolases (Subtilisin,Butelase2,Trypsin,HRV-3C) and the
Aqualigase Variant with a Water-Shielding Pocket Design.</p>
        					</div>

<br>
							<a name= 'Ligase'><h2>Engineering a Ligase Requires Additional Information</h2></a>
							<br>
							<p>
							To evaluate the catalytic activity of the wild-type (WT) and mutant enzymes (19, 105, 143), we
sought to measure their performance in a transpeptidation (ligation) reaction. Reactions were
conducted in the presence of potential nucleophiles: a peptide (32GITRR, at a 32:1 molar ratio to the
substrate) and an amino acid (1000Gly, at a 1000:1 molar ratio). Notably, HPLC analysis revealed no
detectable formation of the expected ligation product in any of the tested conditions. The only
observed product corresponded to the hydrolytic cleavage of the substrate. Consequently, our
analysis focused on the hydrolytic activity of the enzymes under these different conditions,
quantified as the ratio of the hydrolytic product peak area to the remaining substrate peak area. The
results after 12 and 24 hours of reaction are summarized in <a href="#t1">Table 1</a>.
								<br>
	 <div class="table-container">
        <table>
            <thead>
                <tr>
                    <th>Enzyme</th>
                    <th>Condition</th>
                    <th>Hydrolysis Ratio (12h)</th>
                    <th>Hydrolysis Ratio (24h)</th>
                </tr>
            </thead>
            <tbody>
                <!-- WT 酶类型组 -->
                <tr>
                    <td rowspan="3" class="enzyme-type">WT</td>
                    <td>Pure Hydrolysis</td>
                    <td>4.882871</td>
                    <td>5.870103</td>
                </tr>
                <tr>
                    <td>32GITRR</td>
                    <td>6.287923</td>
                    <td>7.013444</td>
                </tr>
                <tr class="enzyme-group-end">
                    <td>1000Gly</td>
                    <td>9.710145</td>
                    <td>8.505435</td>
                </tr>
                
                <!-- Design1 酶类型组 -->
                <tr>
                    <td rowspan="3" class="enzyme-type">Design1</td>
                    <td>Pure Hydrolysis</td>
                    <td>4.831418</td>
                    <td>6.653915</td>
                </tr>
                <tr>
                    <td>32GITRR</td>
                    <td>4.254176</td>
                    <td>6.278571</td>
                </tr>
                <tr class="enzyme-group-end">
                    <td>1000Gly</td>
                    <td>7.637177</td>
                    <td>8.229491</td>
                </tr>
                
                <!-- Design2_IM 酶类型组 -->
                <tr>
                    <td rowspan="3" class="enzyme-type">Design2_IM</td>
                    <td>Pure Hydrolysis</td>
                    <td>6.467882</td>
                    <td>7.574692</td>
                </tr>
                <tr>
                    <td>32GITRR</td>
                    <td>5.53022</td>
                    <td>7.127273</td>
                </tr>
                <tr class="enzyme-group-end">
                    <td>1000Gly</td>
                    <td>5.215517</td>
                    <td>7.909449</td>
                </tr>
                
                <!-- Design2 酶类型组 -->
                <tr>
                    <td rowspan="3" class="enzyme-type">Design2</td>
                    <td>Pure Hydrolysis</td>
                    <td>5.21548</td>
                    <td>12.1319</td>
                </tr>
                <tr>
                    <td>32GITRR</td>
                    <td>4.037259</td>
                    <td>7.263775</td>
                </tr>
                <tr class="enzyme-group-end">
                    <td>1000Gly</td>
                    <td>6.441385</td>
                    <td>8.73125</td>
                </tr>
            </tbody>
        </table>
		    <div class="caption">
            <p><strong><a name="t1">Table 1. </a></strong>Hydrolytic activity of WT and mutant enzymes in the presence of different nucleophiles.
Values represent the ratio of the peak area of the hydrolysis product to the peak area of the
remaining substrate, as determined by HPLC. “Pure Hydrolysis” serves as the control. “32GITRR”
and “1000Gly” represent reactions supplemented with the GITRR peptide (peptide:substrate ratio of
32:1) and glycine (glycine:substrate ratio of 1000:1), respectively. All reactions were conducted under
identical conditions.</p>
        	</div>
    </div>


<br>

<p>
In our pursuit of engineering HRV-3C wild-type (WT) and its variants into efficient peptide
ligases, we assayed their catalytic activities in the presence of potential external nucleophiles—the
GITRR peptide and glycine. Contrary to our design objective, the desired ligation product was not
detected under the tested aqueous conditions, with its concentration remaining consistently below
the limit of detection of the HPLC method. Instead, the predominant catalytic activity observed was
substrate hydrolysis. Consequently, our investigation pivoted to a systematic characterization of
how these additives influence the intrinsic hydrolytic function of the enzymes. The activity,
quantified by the peak area ratio of the hydrolysis product to the remaining substrate, is presented
in Table 1. In the control (pure hydrolysis), mutant Design2 was identified as the most hydrolytically
active variant, exhibiting a 24-hour activity ratio (12.13) markedly higher than that of the WT (5.87) and other mutants. Intriguingly, rather than facilitating ligation, these nucleophilic compounds
served as modulators of hydrolytic activity in an enzyme-specific manner. For the highly active
mutant Design2, both 32GITRR and 1000Gly exerted an inhibitory effect, reducing its activity ratio
to 7.26 and 8.73, respectively. In sharp contrast, the WT enzyme was consistently activated by these
additives, with its 24-hour activity ratio increasing from 5.87 to 7.01 and 8.50 in the presence of
32GITRR and 1000Gly, respectively. These findings suggest that despite their inefficacy as ligation
substrates, these molecules can interact with the enzyme and allosterically modulate its canonical
hydrolytic pathway.
							</p>
							<br>


							<a name= 'Dis'><h2>Discussions</h2></a>
							<br>
<p>
The strategy of rationally designing the HRV-3C protease to construct its hydrophobic pocket for
developing novel ligases stems from a thorough evaluation of existing mainstream ligase
technologies. First, PAL family ligases derived from asparaginyl endopeptidase (represented by
Butelase 1) face bottlenecks in expression and purification. Despite their high catalytic efficiency and
short recognition sequence (the tripeptide motif D/N-HV), these enzymes originate from the plant
butterfly pea, making recombinant expression challenging[4]. The complex activation process
required for their proenzyme form is difficult to precisely control, affecting experimental
reproducibility. While the short recognition sequence facilitates handling, it increases the risk of
non-specific reactions. Practical implementation also requires high substrate concentrations to drive
the reaction, potentially triggering substrate inhibition and escalating costs. Additionally, Subtiligase
derived from Bacillus subtilis protease faces constraint: substrate pretreatment requirements.[27]
While this enzyme demonstrated the feasibility of serine protease-catalyzed ligation, it requires preactivation
of the substrate C-terminus into a high-energy ester. This additional step complicates
operations, increases costs, and may induce side reactions like racemization, limiting its application
in peptide modification. Finally, Staphylococcus aureus Sortase A suffers from core defects:
thermodynamic reversibility and low kinetic efficiency. Its reversible catalytic mechanism limits conversion rates, necessitating excessive use of donor substrates, which increases costs and
purification complexity. The reaction exhibits sensitivity to conditions and poor reproducibility,
making standardized workflow establishment challenging[28]. Slow catalytic rates often necessitate
high substrate concentrations, compromising economic viability. Its application in cyclic peptide
synthesis is particularly limited due to the generation of lengthy “scar” sequences and the
requirement for substrates exceeding 19 amino acids to avoid oligomerization side reactions. In
summary, existing ligases exhibit significant shortcomings in expression complexity, substrate
pretreatment requirements, and reaction reversibility. Consequently, there is an urgent need to
develop novel ligases that can be efficiently expressed in E. coli, require no preactivation, directly
utilize native carboxylate substrates, and exhibit irreversible reactions. The HRV-3C protease offers
advantages such as high prokaryotic expression yield, excellent stability, and strong specificity
(recognizing the long sequence LEVLFQ-G), making it an ideal starting point for engineering.
Through rational design to construct a hydrophobic pocket, its binding capacity with hydrophobic substrates can be enhanced, optimizing the reaction pathway and thereby achieving a functional
transformation from a hydrolase to an efficient ligase.
</p>

				<table>
                    <thead>
                        <tr>
                            <th>Reaction</th>
                            <th>System</th>
                            <th>Environment</th>
                            <th>Acyl donor</th>
                            <th>Acyl acceptor</th>
                        </tr>
                    </thead>
                    <tbody>
                        <!-- Hydrolytic reaction group -->
                        <tr>
                            <td rowspan="5" class="reaction-type">Hydrolytic</td>
                            <td>Subtilisin</td>
                            <td>pH8, 25°C</td>
                            <td>Suc-AAPF</td>
                            <td>---</td>
                        </tr>
                        <tr>
                            <td>Aqualigase</td>
                            <td>pH8, 40°C</td>
                            <td>Suc-AAPF</td>
                            <td>---</td>
                        </tr>
                        <tr>
                            <td>HRV-3C</td>
                            <td>pH8, 4°C</td>
                            <td>LEVLFQ</td>
                            <td>---</td>
                        </tr>
                        <tr>
                            <td>Butelse2</td>
                            <td>pH7, 37°C</td>
                            <td>ISYRN</td>
                            <td>---</td>
                        </tr>
                        <tr class="enzyme-group-end">
                            <td>Trypsin</td>
                            <td>pH8, 37°C</td>
                            <td>GGGY</td>
                            <td>---</td>
                        </tr>
                        
                        <!-- Ligation reaction group -->
                        <tr>
                            <td rowspan="5" class="reaction-type">Ligation</td>
                            <td>Aqualigase</td>
                            <td>pH8, 40°C</td>
                            <td>Suc-AAPF</td>
                            <td>AF-NH2</td>
                        </tr>
                        <tr>
                            <td>Subtiligase</td>
                            <td>pH8, 25°C</td>
                            <td>Suc-AAPF</td>
                            <td>AF-NH2</td>
                        </tr>
                        <tr>
                            <td>HRV-3C</td>
                            <td>pH8, 4°C</td>
                            <td>LEVLFQ</td>
                            <td>GITRR</td>
                        </tr>
                        <tr>
                            <td>Bu2g(V/GA)</td>
                            <td>pH7, 37°C</td>
                            <td>GISTKSIPPISYRN</td>
                            <td>GISTKSIPPISYRN</td>
                        </tr>
                        <tr class="enzyme-group-end">
                            <td>Trypsiligase</td>
                            <td>pH8, 37°C</td>
                            <td>GGGY</td>
                            <td>RNGGG</td>
                        </tr>
                    </tbody>
                </table>
                
                <div class="caption">
                    <p><strong><a name= 't2'></a>Table 2.</strong> Systems and construction conditions used in this study</p>
                </div>

<br>
<p>
Predicated on the assumption that a portion of the mutant proteins retained correct folding, the
lack of observed ligation activity indicates that the strategy of constructing a “water-proof pocket”
solely through the introduction of hydrophobic residues is inadequate to surmount the potent
hydrolytic background. Water acts as a ubiquitous and highly competitive nucleophile in aqueous
environments, with an effective concentration reaching approximately 55 M. Furthermore, the active
sites of natural hydrolases have undergone extensive evolutionary optimization to exploit this
abundance efficiently. While our cMD simulations revealed that the distribution of water attack
angles in the mutants deviated from that of the WT, this alteration was likely insufficient to
fundamentally preclude the hydrolytic reaction. Consequently, dismantling this evolutionarily
refined “hydrolysis machinery” may necessitate structural remodeling that is more drastic or
intricate than previously envisioned.
</p>
<br>
<p>
This insight pivots our strategic focus for future designs. Our initial design of the hydrophobic
pocket prioritized the direct exclusion of water; however, future work should specifically target the
stage of competition with hydrolysis: the attack by the peptide nucleophile substrate. Characterizing
the kinetics of this peptide’s attack will provide a direct and far more meaningful metric for
assessing our progress in creating a truly competitive ligation pathway.
</p>

<a name= 'Bibli'><h2>Bibliography</h2></a>
<p>
<br>
[12] M. Tiberti et al., “MutateX: an automated pipeline for in silico saturation mutagenesis of
protein structures and structural ensembles,” Briefings in bioinformatics, vol. 23, no. 3, p.
bbac74, 2022.
<br>
[13] J. Abramson et al., “Accurate structure prediction of biomolecular interactions with AlphaFold
3,” Nature, vol. 630, no. 8016, pp. 493–500, 2024.
<br>

[14] O. Trott and A. J. Olson, “AutoDock Vina: Improving the speed and accuracy of docking with a
new scoring function, efficient optimization, and multithreading,” Journal of Computational
Chemistry, vol. 31, no. 2, pp. 455–461, 2010, doi: https://doi.org/10.1002/jcc.21334.
<br>

[15] T. Lu, “A comprehensive electron wavefunction analysis toolbox for chemists, Multiwfn,” The
Journal of Chemical Physics, vol. 161, no. 8, p. 82503, 2024, doi: 10.1063/5.0216272.
<br>

[16] T. Lu, “Tian Lu, Sobtop, Version 1.0 (dev 5) http://sobereva.com/soft/Sobtop (accessed on 2
10,2025),” vol. 0, p. , 2025.
<br>

[17] M. J. Abraham et al., “GROMACS: High performance molecular simulations through multilevel
parallelism from laptops to supercomputers,” SoftwareX, pp. 19–25, 2015, doi: https://doi.
org/10.1016/j.softx.2015.06.001.
<br>

[18] C. Tian et al., “ff19SB: Amino-Acid-Specific Protein Backbone Parameters Trained against
Quantum Mechanics Energy Surfaces in Solution,” Journal of Chemical Theory and
Computation, vol. 16, no. 1, pp. 528–552, 2020, doi: 10.1021/acs.jctc.9b00591.
<br>

[19] S. Izadi, R. Anandakrishnan, and A. V. Onufriev, “Building Water Models: A Different
Approach,” The Journal of Physical Chemistry Letters, vol. 5, no. 21, pp. 3863–3871, 2014, doi:
10.1021/jz501780a.
<br>

[20] D. A. Case et al., “AmberTools,” Journal of Chemical Information and Modeling, vol. 63, no. 20,
pp. 6183–6191, 2023, doi: 10.1021/acs.jcim.3c01153.
<br>

[21] J. Wang and Y. Miao, “Ligand Gaussian Accelerated Molecular Dynamics 3 (LiGaMD3):
Improved Calculations of Binding Thermodynamics and Kinetics of Both Small Molecules and
Flexible Peptides,” Journal of Chemical Theory and Computation, vol. 20, no. 14, pp. 5829–5841,
2024, doi: 10.1021/acs.jctc.4c00502.
<br>

[22] Z. Zhang, W. X. Shen, Q. Liu, and M. Zitnik, “Efficient generation of protein pockets with
PocketGen,” Nature Machine Intelligence, pp. 1–14, 2024.
<br>

[23] Z. Lin et al., “Evolutionary-scale prediction of atomic-level protein structure with a language
model,” Science, vol. 379, no. 6637, pp. 1123–1130, 2023, doi: 10.1126/science.ade2574.
<br>

[24] M. S. Valdés-Tresanco, M. E. Valdés-Tresanco, P. A. Valiente, and E. Moreno, “gmx_MMPBSA:
A New Tool to Perform End-State Free Energy Calculations with GROMACS,” Journal of
Chemical Theory and Computation, vol. 17, no. 10, pp. 6281–6291, 2021, doi: 10.1021/
acs.jctc.1c00645.
<br>

[25] J. L. Watson et al., “De novo design of protein structure and function with RFdiffusion,” Nature,
vol. 620, no. 7976, pp. 1089–1100, 2023.
<br>

[26] J. Dauparas et al., “Atomic context-conditioned protein sequence design using LigandMPNN,”
Biorxiv, pp. 2023–2012, 2023.
<br>

[27] T. Nuijens, A. Toplak, M. Schmidt, A. Ricci, and W. Cabri, “Natural occurring and engineered
enzymes for peptide ligation and cyclization,” Frontiers in Chemistry, vol. 7, p. 829, 2019.
<br>

[28] R. Warden-Rothman, I. Caturegli, V. Popik, and A. Tsourkas, “Sortase-Tag Expressed Protein
Ligation: Combining Protein Purification and Site-Specific Bioconjugation into a Single Step,”
Analytical Chemistry, vol. 85, no. 22, pp. 11090–11097, 2013, doi: 10.1021/ac402871k.
							</p>
						</div>

					</div>
				</div>
			</div>
		</div>
	</div>

<!-- End Service Section  -->
<!--==================================================-->

<!-- =====================footer section start======================== -->
<div class="footer-section">
	<div class="container">
		<div class="row">
			<div class="col-lg-2 col-md-6 col-sm-12">
				<div class="footer-image">
					<a href=""><img src="image/logo_transp.png" alt=""></a>
				</div>
				<h2 style="font-size: 16px">iDEC 2025|CPU_CHINA </h2>
				<div class = 'bar1'></div>
				<h2 style="font-size: 16px; margin-bottom: 20px;">Sponsor</h2>
				<div class="footer-image">
					<a href=""><img src="image/fulexi.jpg" alt=""></a>
				</div>
				<!--
				<div class="footer-about-social-icon pt-40">
					<ul>
						<li>
							<a href="#"><i class="fab fa-facebook-f"></i></a>
						</li>
						<li>
							<a href="#"><i class="fab fa-pinterest"></i></a>
						</li>
						<li>
							<a href="#"><i class="fab fa-linkedin-in"></i></a>
						</li>
					</ul>
				</div>
				-->
			</div>
			<div class="col-lg-4 col-md-6 col-sm-12" >
				<h2>About Our Project</h2>
				<p>
					This
study aimed to convert the HRV-3C protease, a highly specific cysteine hydrolase, into a novel
peptide ligase through rational design. Our primary strategy involved computationally engineering
a hydrophobic active-site pocket to suppress the enzyme’s intrinsic hydrolytic activity by sterically
hindering the access of nucleophilic water molecules.<br>
				</p>
			</div>
			
			<div class="col-lg-4 col-md-6 col-sm-12" >
			<h2>China Pharmaceutical University</u></h2>
				<div class="footer-icon d-flex align-items-center">
					<div class="icon">
						<i class="fa fa-map-marker" aria-hidden="true"></i>
					</div>
					<p>Jiangning Campus： #639 Longmian Dadao, Jiangning District, Nanjing, China，211198</p>
				</div>

				<div class="bar2"></div>

				<div class="footer-icon d-flex align-items-center">
					<div class="icon">
						<i class="fa fa-map-marker" aria-hidden="true"></i>
					</div>
					<p>Xuanwumen Campus：#24 TongJiaXiang, Gulou District, Nanjing, China，210009</p>
				</div>
			</div>
			
			<div class="link col-lg-2 col-md-6 col-sm-12" >
				<h2>Useful Links</h2>
				<ul>
					<li><a href="background.html"><i class="fa fa-angle-double-right" aria-hidden="true"></i>Background</a></li>
					<li><a href="result.html"><i class="fa fa-angle-double-right" aria-hidden="true"></i>Result</a></li>
					<li><a href="files/idec25.pdf"><i class="fa fa-angle-double-right" aria-hidden="true"></i>Report</a></li>
					<li><a href="files/idec25_si.pdf"><i class="fa fa-angle-double-right" aria-hidden="true"></i>Supplementary Information</a></li>
				</ul>
			</div>
			
		</div>
		
	</div>
	<div class="footer-bar"></div>
	<div class="cuppyright-section">

		<p>iDEC 2025 | CPU_CHINA <a target="_blank" ></a></p>
		
	</div>
</div>


<!-- ====================footer section end========================= -->

<!--==================================================-->

	<!-- loder -->
	 <!--
	<div class="loader-wrapper">
		<div class="loader"></div>
		<div class="loder-section left-section"></div>
		<div class="loder-section right-section"></div>
	</div>
	 -->

<!-- scroll strat============  -->

<div class="scroll-area">
	<div class="top-wrap">
		<div class="go-top-btn-wraper">
			<div class="go-top go-top-button">
				<i class="fas fa-arrow-up"></i>
				<i class="fas fa-arrow-up"></i>
			</div>
		</div>
	</div>
</div>

<!-- scroll end============  -->

<!-- scroll 2 strat============  -->
	<!-- <div id="progress" class="progress hide">
		<div id="progress-value"></div>
	</div> -->
<!-- scroll end============  -->

<!-- Start Search Popup Area -->
<!--==================================================-->
<!--
<div class="search-popup">
	<button class="close-search style-two"><i class="fa fa-times" aria-hidden="true"></i></button>
	<button class="close-search"><i class="fa fa-arrow-up" aria-hidden="true"></i></button>
	<form method="post" action="#">
		<div class="form-group">
			<input type="search" name="search-field" value="" placeholder="Search Here" required="">
			<button type="submit"><i class="fa fa-search" aria-hidden="true"></i></button>
		</div>
	</form>
</div>
-->

<!--==================================================-->

<!-- jquery js -->	
<script src="js/jquery-3.2.1.min.js"></script>
<!-- bootstrap js -->	
<script src="js/bootstrap.min.js"></script>
<!-- carousel js -->
<script src="js/owl.carousel.min.js"></script>
<!-- counterup js -->
<script src="js/jquery.counterup.min.js"></script>
<!-- waypoints js -->
<script src="js/waypoints.min.js"></script>
<!-- wow js -->
<script src="js/wow.js"></script>
<!-- imagesloaded js -->
<script src="js/imagesloaded.pkgd.min.js"></script>
<!-- venobox js -->
<script src="js/venobox.js"></script>
<!-- <script src="assets/js/tilt.jquery.min.js"></script> -->
<!-- ajax mail js -->
<script src="js/ajax-mail.js"></script>
<!--  animated-text js -->	
<script src="js/animated-text.js"></script>
<!-- venobox min js -->
<script src="js/venobox.min.js"></script>
<!-- isotope js -->
<script src="js/isotope.pkgd.min.js"></script>
<!-- jquery nivo slider pack js -->
<script src="js/jquery.nivo.slider.pack.js"></script>
<!-- jquery meanmenu js -->	
<script src="js/jquery.meanmenu.js"></script>
<script src="js/popper.min.js"></script>
<!-- jquery scrollup js -->	
<script src="js/jquery.scrollUp.js"></script>
<!-- Jquery UI js -->
<!-- <script  src="assets/js/jquery-ui.min.js"></script> -->
<!-- scrollCue -->
<script src="js/scrollCue.min.js"></script>

<script src="#"></script>
<!-- jquery js -->	
<script src="js/main.js"></script>


<script type="text/javascript">
	
	window.addEventListener('wheel', function(event) {
	    if (event.deltaY < 0) {
	        var page = window.pageYOffset;
	        if(page < 250)
	        {
	        	$(".nav-sidebar").css({
				    'top': '',
				    'position': '',
				    'left': ''
				});

	        }
	    } else if (event.deltaY > 0) {
	        // 鼠标向下滚动

	        var page = window.pageYOffset;
	        if(page >= 250)
	        {
	        	$(".nav-sidebar").css({
				    'top': '125px',
				    'position': 'fixed',

				});
	        }
	        console.log($(".nav-sidebar"));
	    	
	        console.log('鼠标向下滚动');
	    }
	});



</script>

</body>
</html>